### PR TITLE
INT-2233: Updated README.md and added CDN example

### PR DESCRIPTION
**Pull request by tiny-james**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,20 +1,19 @@
 
-# TinyMCE Web Component
+# Official TinyMCE Web Component
```
Comment (jscasca) [inline README.md:2]: Are we?

**Review by jscasca — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (tiny-james) [reply README.md:2]: An interesting question. I would assume anything publicly released by Tiny is "official". I mainly copied the title structure from the react repo.

Comment (EkimChau) [reply README.md:2]: its the official preview of whats to come, full wc support  :)

Comment (tiny-james) [reply README.md:2]: So should I stick "Preview" in the title? "Official TinyMCE Web Component Preview"?

**Review by EkimChau — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (lnewson) [reply README.md:2]: If it's a preview, then it shouldn't be version `1.0` and should either be `0.x` or a prerelease though.

File: src/demo/html/cdn.html [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+<html>
+  <head>
+    <meta charset="utf-8"/>
+    <title>TinyMCE WebComponent Simple CDN Demo Page</title>
+    <link rel="icon" href="data:;base64,iVBORw0KGgo=">
+    <script src="https://cdn.jsdelivr.net/npm/@tinymce/tinymce-webcomponent@1/dist/tinymce-webcomponent.js"></script>
+  </head>
+
+  <body>
+  <h1>TinyMCE Web Component Simple CDN Demo Page</h1>
```
Comment (lnewson) [inline src/demo/html/cdn.html:11]: Not sure if you're worried, but this doesn't match the title above (I believe the title is a typo).

Comment (tiny-james) [reply src/demo/html/cdn.html:11]: The same typo was on all the demo pages!

**Pull request merged**